.travis.yml

```diff
@@ -2,18 +2,19 @@ sudo: false
 language: php
 
 php:
-    - 5.4
-    - 5.5
```

Comment:
These two versions are super old (and don't typically work on travis anymore), so I removed them.